### ISSUE TITLE
Need pretrained_Duke.pth

### ISSUE THREAD

**Issue by xi-mao** [KovenYu/MAR#34]
If i regard Duke as source dataset to train, I need pretrained model of duke. Is there the code to train the pretraind model?

**Comment by KovenYu** [comment]
please refer to [this issue](https://github.com/KovenYu/MAR/issues/13)

**Comment by pzhren** [comment]
> If i regard Duke as source dataset to train, I need pretrained model of duke. Is there the code to train the pretraind model?

Hello, I want to ask if your problem is solved. KovenYu/MAR#13
I don't know how to do pre-training. I asked the original author but I haven't got a reply yet, I look forward to your help.
The following are the hyperparameter settings during pre-training.
python version : 3.5.4 |Continuum Analytics, Inc.| (default, Aug 14 2017, 13:26:58) [GCC 4.4.7 20120313 (Red Hat 4.4.7-1)]
torch version : 1.1.0

------------------------------------------------------- options --------------------------------------------------------
batch_size: 256 beta: 0.2 crop_size: (384, 128)
epochs: 60 gpu: 0 img_size: (384, 128)
lamb_1: 0.0002 lamb_2: 50.0 lr: 0.001
margin: 1.0 mining_ratio: 0.005 ml_path: ../data/ml_Market.dat
padding: 7 pretrain: True pretrain_path: ../data/resnet50-19c8e357.pth
print_freq: 100 resume: save_path: ../runs/debug
scala_ce: 30.0 source: MSMT17 target: Market
wd: 0.025
https://github.com/pzhren/Papers/blob/master/%E7%9B%AE%E6%A0%87%E6%A3%80%E6%B5%8B%E4%B8%8Ere-id%E4%BB%BB%E5%8A%A1/MAR-master/src/pretrain.py#L6
